Continued migration to the centralized repostitory (#149)
tektoncd/website: tutorials/katacoda/getting-started/setup.md

## Before you begin

Start an experimental Kubernetes cluster in Katacoda with command
`launch.sh`{{execute}}.

Run `kubectl cluster-info`{{execute}} in the terminal to check its status.
You should see the following outputs:

```
Kubernetes master is running at ...
KubeDNS is running at ...

...
```

## Installing Tekton

To add Tekton to this experimental Kubernetes cluster, execute the command
below:

> `kubectl apply --filename https://storage.googleapis.com/tekton-releases/latest/release.yaml`{{execute}}

Reviewer: If we truly want the latest we need to update the URL as this old structure is no longer used and points at Pipelines 0.10.

Updated URL (currently pointing to 0.13.2) is:

```suggestion
`kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml`{{execute}}
```

It may take a few moments for the installation to complete. To monitor the
progress, run the following command:

`kubectl get pods --namespace tekton-pipelines`{{execute}}

Every component listed in the output should have the status `running`.

## Getting the code

We’ve put everything you need for this scenario in a GitHub repository.
To clone this repository, run the following command:

`git clone https://github.com/tektoncd/website`{{execute}}

> Open the directory `tekton-example/getting-started/src` with the file explorer.

Reviewer: ```suggestion
Open the directory `website/tutorials/getting-started/src` with the file explorer.
```

There are a few other references to the path in other files that also need to be updated.

The directory consists of three subdirectories and one file: 

* `app/`: a simple Python Flask web application.
* `tekton-katacoda/`: Tekton resource specifications you will use in this scenario.

> * `tekton-qwiklabs/`: N/A for this scenario.

Reviewer: Can we remove the qwiklabs content and any references to it? It's not currently used and just distracts from the katacoda scenario content.

* `Dockerfile`: a Dockerfile for building app/ into a runnable container image.

## Almost done

Tekton is now running in your Katacode experimental cluster. To help the
installation run smoothly in this special environment, a few extra steps
are required:

```sh
mkdir /mnt/data && kubectl apply -f https://k8s.io/examples/pods/storage/pv-volume.yaml && \

> kubectl apply -f ~/tutorials/katacoda/getting-started/src/tekton-katacoda/init.yaml && \

Reviewer: ```suggestion
kubectl apply -f ~/website/tutorials/katacoda/getting-started/src/tekton-katacoda/init.yaml && \
```
